api/scripts/dev/get-latest-release.js

```diff
@@ -0,0 +1,68 @@
+const dotenv = require('dotenv');
+dotenv.config();
+const fs = require('fs');
+const { performance } = require('perf_hooks');
+const { disconnect } = require('../../db/knex-database-connection');
+const releaseRepository = require('../../lib/infrastructure/repositories/release-repository');
+const logger = require('../../lib/infrastructure/logger');
+const argv = require('yargs/yargs')(process.argv.slice(2))
+  .version(false)
+  .usage('Usage: $0 [options]')
+  .option('file', {
+    alias: 'f',
+    describe: 'Destination file',
+    nargs: 1,
+    requiresArg: true,
+  })
+  .option('createRelease', {
+    alias: 'c',
+    describe: 'Create a release beforehand',
+    requiresArg: false,
+  })
+  .demandOption(['file'], 'Please provide a destination file to dump the release into')
+  .example('$0 -f release.txt', 'Dump the latest release in the given destination file')
+  .example('$0 -f release.txt --createRelease', 'Create a release then dump it in the given destination file')
+  .help('h')
+  .alias('h', 'help')
+  .argv;
+
+const getLatestRelease = async ({ shouldCreateRelease, destFile }) => {
+  if (shouldCreateRelease) {
+    logger.info('Creating release...');
+    await releaseRepository.create();
+    logger.info('Release created.');
+  }
+  logger.info('Fetching latest release...');
+  const release = await releaseRepository.getLatestRelease();
+  fs.writeFileSync(destFile, JSON.stringify(release));
```

Comment:
Je ne sais pas si c'est le cas, mais si on veut que le fichier soit formatté on peut faire :
```suggestion
  fs.writeFileSync(destFile, JSON.stringify(release, null, 2));
```

Reply:
ok je vais voir ce que ça fait exactement pour apprendre et je fais, merci

Reply:
Ok je le fais pas car a priori je m'en fiche qu'il soit formatté vu que c'est un pavé de 10 mega octets à traiter de préférence avec un outil style jq

Reply:
donc je m'en fiche que l'espacement soit propre à mon avis